src/v/security/authorizer.h

```diff
@@ -58,28 +58,7 @@ struct auth_result {
     // If found, the role that was matched to provide authZ decision
     std::optional<security::role_name> role;
 
-    friend std::ostream& operator<<(std::ostream& os, const auth_result& a) {
```

Comment:
Now you can remove `<fmt/core.h>` to the `.cc`, but you'll have to add `<iosfwd>`